assets/stylesheets/admin/style.css

```diff
@@ -2213,4 +2213,12 @@ body .swal2-animate-x-mark {
   margin-top: -3px;
 }
 
+.message.warning {
```

Comment:
Ailsan, nós sempre adicionamos o pre-fixo pagar-me nas classes de CSS para evitar conflitos com temas e o padrão do Wordpress. O nome sugerido para sua classe é bastante genérico, logo, uma sugestão:

```suggestion
.pagar-me-message-warning {
```

Reply:
@dias-joao alterei o nome do css para pagar-me-message-warning e aproveitei para modificar a nomenclatura do método que checa se é modo sandbox para o padrão woocommerce. Commit e41a9cb